### ISSUE TITLE
Steam deck oled. wont install

### ISSUE THREAD

**Issue by Jord3030** [moraroy/NonSteamLaunchers-On-Steam-Deck#152]
Followed the steps on here and the youtube videos.   when i run the installer, somtimes the konsole starts somtimes it doesnt, but it will never run any sort of scrpit or download anything.

**Comment by moraroy** [comment]
@Jord3030 send the log it's in the downloads folder

**Comment by Jord3030** [comment]
[NonSteamLaunchers-install.log](https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/files/13631856/NonSteamLaunchers-install.log)

**Comment by Jord3030** [comment]
its now downloading  kinda.   but just closes near the end not doing anything

**Comment by Jord3030** [comment]
lolol log says its installed but its nowhere to be found. also no sign of steam closing until after its done

**Comment by Jord3030** [comment]
@moraroy. My konsole looks nothing like in the videos. Just says nohup.out. and nothing else.   Then the download for Uplay jumps straight to halfway and stays there then closes everything.  Then steam will close and reopen.  Nothing is happening after that.
![20231211_004152](https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/assets/153482225/fbe595a3-c057-4cf7-9f9b-d8aa143ee752)

**Comment by Jord3030** [comment]
@moraroy i do not see ubisoft connect in mu library.

**Comment by moraroy** [comment]
@Jord3030 do you have multiple accounts on your Steam deck?

**Comment by moraroy** [comment]
Do me a favor @Jord3030 press the start fresh button and try and download your launchers 1 by 1

**Comment by Jord3030** [comment]
@moraroy i have done that 6 times now
redownloaded the client afterwards each time too. start fresh and uninstall doesnt do anything.  once starting it back up it does the exact same thing, jumps to 50% with the konsole not saying anything, then closing,

**Comment by moraroy** [comment]
@Jord3030 that's strange I've never ran into that issue... check inside the compatdata folder after you have clicked Start Fresh is there any symlinks left over or NonSteamLaunchers or anything? Are you trying to put it on the SD card?

**Comment by Jord3030** [comment]
@moraroy do u have a discord or something we could speak on?

**Comment by Jord3030** [comment]
jord3030

**Comment by AllPowerfulNick** [comment]
Hello there, I've run into this same issue on my 1tb oled. What was the verdict?

**Comment by moraroy** [comment]
@AllPowerfulNick are you running the .desktop file from the desktop itself?

**Comment by AllPowerfulNick** [comment]
@moraroy I tried it both ways, from the extract folder and from from the .desktop app

**Comment by moraroy** [comment]
@AllPowerfulNick yeah but what I'm saying is did you run the desktop app on the actual Desktop.

**Comment by AllPowerfulNick** [comment]
@moraroy yes, after following the guide from your link, console didn't run anything

**Comment by moraroy** [comment]
@AllPowerfulNick hit "Start Fresh" in NSL and then go into the compatdata folder and check and make sure there are no symlinks or anything pertaining to Nonstaeamlaunchers. If there are delete them.

**Comment by AllPowerfulNick** [comment]
Took me a bit to find the folder, but I do not see anything NSL in compatdata or elsewhere
